app/Http/Controllers/CategoryController.php

```diff
@@ -56,7 +75,22 @@ public function show($id)
      */
     public function edit($id)
     {
-        //
+        $category = Category::findOrFail($id);
+        $categoryList = Category::all();
+
+        if ($category->parent_id === config('number_format.parent_id')){
+            $category->load('children');
+
+            return view('website.backend.category.update', [
+                'category' => $category,
+                'categoryList' => $categoryList,
+            ]);
```

Comment:
có thể dử dụng như sau
```suggestion
            return view('website.backend.category.update', compact('category', 'categoryList'));
```